default.nix

```diff
@@ -93,6 +93,7 @@ let #TODO: Upstream
     addLibs = self: super: {
       obelisk-asset-manifest = self.callCabal2nix "obelisk-asset-manifest" (hackGet ./lib/asset + "/manifest") {};
       obelisk-asset-serve-snap = self.callCabal2nix "obelisk-asset-serve-snap" (hackGet ./lib/asset + "/serve-snap") {};
+      cliapp = self.callCabal2nix "cliapp" (cleanSource ./lib/cliapp) {};
```

Comment:
This should be named `obelisk-cliapp`.  That naming convention should apply to everything in the `lib` directory.

Let's also add a README.md in that directory explaining this convention.